src/archivist/fetcher/index.js

```diff
@@ -1,23 +1,41 @@
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+import config from 'config';
+
 import fetchFullDom from './fullDomFetcher.js';
 import fetchHtmlOnly from './htmlOnlyFetcher.js';
 
 export { launchHeadlessBrowser, stopHeadlessBrowser } from './fullDomFetcher.js';
 export { FetchDocumentError } from './errors.js';
 
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const defaultConfigs = JSON.parse(fs.readFileSync(path.resolve(__dirname, '../../../config/default.json')));
+
+config.util.setModuleDefaults('fetcher', defaultConfigs.fetcher);
```

Comment:
> This is great! But maybe a bit heavy loading hidden deep in a module `index`. Shouldn't we set this module default in the main config setup?

Reply:
I don't see how we can do this, because it has to be loaded even when a reuser import only this module.